api/controllers/healthcheck.js

```diff
@@ -13,14 +12,13 @@ exports.db = async (req, res) => {
     conn = await req.app.locals.pool.getConnection();
     const q = 'SELECT 1 as ok FROM Track WHERE ID=1 LIMIT 1';
     const result = await conn.query(q);
-
     result[0].ok = result[0].ok === 1;
-    // `${res.json(result[0])}ok!`;
+    res.json(result[0]);
   } catch (err) {
     res.json(`error${err}`);
-    // lib.error(err, res, 500, false);
   } finally {
     if (conn) {
+      conn.release();
```

Comment:
should be handled by end. should not be needed.

Reply:
ok!